examples/xx-delete-editor-layout-field-set.js

```diff
@@ -0,0 +1,7 @@
+// TODO: rename file before merging to master
```

Comment:
Could those rather be todos on the PR so we see that "x from y task are completed" hint?

Reply:
It makes sense to add that in the final pr targeting master. For now we need to note it somewhere else I think